chunky/src/java/se/llbit/chunky/renderer/scene/AsynchronousSceneManager.java

```diff
@@ -182,13 +182,13 @@ public synchronized void mergeRenderDump(File renderDump) {
    *
    * @return the preferred scene name
    */
-  public static String preferredSceneName(RenderContext context, String name) {
+  public static String preferredSceneName(RenderContext context, File parentDirectory, String name) {
```

Comment:
By adding the scene directory to the `RenderContext`, you could get it from there and keep this method as is.